ui/src/styles.css

```diff
@@ -196,3 +196,19 @@ iframe#launcher {
     transform: rotate(359deg);
   }
 }
+
+/*
+ * Hides the unnecessary "select all" header from the multiselect drop-down for the
+ * Degrees multi-select in the Account Creation component.
+ */
+.create-account__degree-select .p-multiselect-header {
+  display: none;
+}
+
+.terms-of-service__page {
```

Comment:
Our usual pattern for React components is to use inline style (despite my initial grumbling). Is there a need for this to be done in CSS? The classes here don't seem particularly general